packages/react-devtools-extensions/src/injectGlobalHook.js

```diff
@@ -88,7 +88,7 @@ if (sessionStorageGetItem(SESSION_STORAGE_RELOAD_AND_PROFILE_KEY) === 'true') {
 
 // Inject a __REACT_DEVTOOLS_GLOBAL_HOOK__ global for React to interact with.
 // Only do this for HTML documents though, to avoid e.g. breaking syntax highlighting for XML docs.
-if (document.contentType === 'text/html') {
+if (['text/html', 'text/plain'].includes(document.contentType)) {
```

Comment:
I don't think this is the right approach.

Injecting the hook into a plain text file wouldn't serve a purpose, since there's no React script running in such a file. Also it would not fix the problem for other non-HTML document types (e.g. text/json, application/json)

Reply:
I'll make the suggested change later in the day.

Reply:
Sorry, I was on Chinese Lunar Year vocation in the past several days. I'm working on it now...